### PR TITLE
Set chiado deprecated slots to zero

### PR DESCRIPTION
**Pull request by dapplion**
Chiado network deployed for the shapella the contract at commit https://github.com/gnosischain/deposit-contract/commit/13e155500b626612844e3d0fccc11b02b11ea785. This contract version was latter replaced with the contract at commit https://github.com/gnosischain/deposit-contract/pull/45/commits/8fe75d23497d720b12af9e75790a616dfb64ca6b. The latter version leaves some storage slots orphan, which may cause problems in future updates.

This PR introduces a method to set the gap slots to zero. `_deprecated_slots_gap` variable is added once to set storage slots 69,70,71 back to zero 

   
    SBCDepositContract     │           withdrawableAmount           │      67      │                  t_mapping(t_address,t_uint256)                   │      32       │
    SBCDepositContractOld  │ failedWithdrawalIndexByWithdrawalIndex │      68      │                   t_mapping(t_uint64,t_uint256)                   │      32       │
    SBCDepositContractOld  │       numberOfFailedWithdrawals        │      69      │                             t_uint256                             │      32       │
    SBCDepositContractOld  │          nextWithdrawalIndex           │      70      │                             t_uint64                              │       8       │
    SBCDepositContractOld  │        failedWithdrawalsPointer        │      71      │                             t_uint256                             │      32       |

**Deployment plan**
- Upgrade contract to this implementation
- Call `setSlotGapToZero`
- Forget about it

----

- Replaces https://github.com/gnosischain/deposit-contract/pull/46 and https://github.com/gnosischain/deposit-contract/pull/47

### CODE REVIEW

File: contracts/SBCDepositContract.sol [hunk 1]
```diff
@@ -42,6 +42,21 @@ contract SBCDepositContract is
     address private constant SYSTEM_WITHDRAWAL_EXECUTOR = 0xffffFFFfFFffffffffffffffFfFFFfffFFFfFFfE;
     mapping(address => uint256) public withdrawableAmount;
 
+    // failedWithdrawalIndexByWithdrawalIndex
+    // Chiado network deployed for the shapella the contract at commit https://github.com/gnosischain/deposit-contract/commit/13e155500b626612844e3d0fccc11b02b11ea785
+    // This contract version was latter replaced with the contract at commit https://github.com/gnosischain/deposit-contract/pull/45/commits/8fe75d23497d720b12af9e75790a616dfb64ca6b
+    // The latter version leaves some storage slots orphan, which may cause problems in future updates
+    // This variable is added once to set storage slots 69,70,71 back to zero
```
Comment (4rgon4ut) [inline contracts/SBCDepositContract.sol:49]:
```suggestion
    // This variables is added once to set storage slots 69,70,71 and map(68) values back to zero
```

File: contracts/SBCDepositContract.sol [hunk 2]
```diff
@@ -297,4 +312,27 @@ contract SBCDepositContract is
     function unwrapTokens(IUnwrapper _unwrapper, IERC20 _token) external onlyAdmin {
         _unwrapper.unwrap(address(stake_token), _token.balanceOf(address(this)));
     }
+
+    /**
+     * @dev See `_deprecated_slots_gap` comment for rationale
```
Comment (4rgon4ut) [inline contracts/SBCDepositContract.sol:317]:
```suggestion
     * @dev See `_deprecated_slot_X` variables comment for rationale
```

File: contracts/SBCDepositContract.sol [hunk 3]
```diff
@@ -297,4 +309,14 @@ contract SBCDepositContract is
...
+     */
+    function setSlotGapToZero() external onlyAdmin {
```
[the remark this answers is not in the document]
Comment (4rgon4ut) [reply contracts/SBCDepositContract.sol]: Lets just keep as is as we are aiming to remove whole function anyways

**Comment by filoozom** [comment]
This script has been ran to its completion and all slots are now zeroed out on Chiado. Can we close this PR @dapplion?

**Comment by dapplion** [comment]
All slots in chiado have been reset

**Pull request closed without merging**